_posts/2019/2019-08-20-end-to-end-tests-with-geb-and-spock.md

```diff
@@ -0,0 +1,164 @@
+---
```

Comment:
General remarks:

Some of my remarks have not been included. I don't agree with the decision of not including them, as we do have conventions for the developer blog:

* The main language of the Developer Portal is American English (except for the NOW API Docs, that for reasons needed to be written in BE). So there's a general decision and convention of publishing blog posts in AE, and I don't want to make an exception here.
* Same goes for Oxford Commas. We do proof-read "pro"-Oxford comma. And again, here I don't want to make an exception.
* {:target="_blank"} for external links - this is also a convention and written down in the Wiki. Please stick to that.
* Same goes for the way we include images. I've made a comment directly at the line how to do so.

You might have issues with your IDE displaying stuff. But this seems to be an exception as all other DevBlog contributors are fine with how we go along with our conventions.

Remark to the blog post:

The topic is nicely written, thank you for your contribution and effort!
When reading this through, I stumbled over the large reading blocks that are hard to digest. Readers want to scan through a post and at first sight want to decide if they read further. Sub titles and / or shorter sections help digest the text. You might want to think about that.